packages/datastore-storage-adapter/__tests__/ExpoSQLiteDatabase.test.ts

```diff
@@ -244,7 +261,7 @@ describe('ExpoSQLiteDatabase 2', () => {
 		});
 
 		// multiple delete and no save statements
```

Comment:
Nit: Don't really need the comment given the test description
```suggestion
```

Reply:
Got it. Will try to incorporate as much information as possible in the test title.